File: 프로그래머스/unrated/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.js
function solution(n) {
    let result = 0;
    for(let i = 1; i <= n; i++) {
        result += (n % 2 == 1) ? ((i % 2 == 1) ? i : 0) : ((i % 2 == 0) ? i * i : 0);
    }
    return result;
}

File: 프로그래머스/unrated/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.java
class Solution {
    public int solution(int n) {
        int temp = 0;
        if (n % 2 == 1) {
            for (int i = 1; i <= n; i++) {
                if (i % 2 == 1) temp = i + temp;
            }
        } else if (n % 2 == 0) {
            for (int i = 1; i <= n; i++) {
                if (i % 2 == 0) temp = temp + (int)Math.pow(i, 2);
            }
        }
        return temp;
    }
}